canonical-web-and-design/maas-ui
# LP: #1859622 - Fix the console error on networking tab

## Done
Fixed the typo in the send analytics function on the Network tab.

## QA
- Run this branch
- Go into a machine
- Select Network
- Check there is no console error

## Fixes
Fixes https://github.com/canonical-web-and-design/maas-ui/issues/655

## Launchpad Issue
LP: #1859622
